.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [8-jdk, 11-jdk, 15-jdk]
+        java: [8-jdk, 16-jdk]
```

Comment:
i dont see what is wrong with running on java 11

Reply:
See https://github.com/Legacy-Fabric/fabric-installer/pull/1#discussion_r668186745